src/physics/large_eddy_simulation.h

```diff
@@ -51,6 +51,19 @@ class LargeEddySimulationBase : public ModelBase <dim, nstate, real>
         const std::array<dealii::Tensor<1,dim,real>,nstate> &solution_gradient,
         const dealii::types::global_dof_index cell_index) const;
 
+    /// Convective Numerical Split Flux for split form
```

Comment:
Update comment to be consistent with what is in model_base.h

Reply:
Updated the comments based on the current changes